api/WebGLRenderingContext.json

```diff
@@ -37,7 +37,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```

Comment:
(All of the changes in this file should be "1.5" instead, but there's just so many.)

Reply:
Done